src/App.js

```diff
@@ -1,11 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import './components/style.css';
+import Calculate from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
         <p>
           Edit
           <code>src/App.js</code>
```

Comment:
- [ ] As stated by the previous reviewer, kindly ensure that the App.js component is converted into a class based component and not functional since it is a project requirement. 
- [ ] Also there seems to be a bug which is preventing your calculator to display properly from the App.js. Kindly ensure that everything is working fine and it displays as shown below to match the given design

![Screenshot from 2023-01-26 10-17-11](https://user-images.githubusercontent.com/65233860/214812502-3ec7c35c-6d2f-4c4f-818b-efbcb10f85ea.png)